src/components/PageTitle/PageTitle.test.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { render } from "@testing-library/react";
+// import "@testing-library/jest-dom/extend-expect";
+import PageTitle from "./PageTitle";
+
+describe("<PageTitle />", function () {
+  it("should match the snapshot", () => {
+    const { getByTestId } = render(<PageTitle label="Github user details" />);
+
+    const pageTitle = getByTestId("page-title");
+    expect(pageTitle).toMatchSnapshot();
```

Comment:
The test is ok, but we can do it better if you do not get element by id, but just get first child of the container
```javascript
const { container: { firstChild } } = render(...)

expect(fisrtChild).toMatchSnapshot()
```

Also, please, add snapshot file to the Git, because it should be reviewed and merged alongside the tests